backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/ImportDirectoryTask.java

```diff
@@ -4,7 +4,7 @@
 import org.molgenis.emx2.Schema;
 import org.molgenis.emx2.io.tablestore.TableStoreForCsvFilesDirectory;
 
-public class ImportDirectoryTask extends ImportSchemaTask {
+public class ImportDirectoryTask extends ImportSchemaTask<ImportDirectoryTask> {
```

Comment:
so a ImportDirectoryTask is a ImportSchemaTask of type ImportDirectoryTasks hmm, not sure about this

Reply:
me neither, will check ;-)